### PR TITLE
WIP updates to AP and Solar System

### PR DESCRIPTION
**Pull request by ebellm**

### CODE REVIEW

File: alertproduction.tex [hunk 1]
```diff
@@ -55,29 +55,23 @@ \subsection{Supporting Incremental Template Generation}
...
 
-Due to the need to verify the instrument characteristics, template quality, and image differencing and Real/Bogus performance, real-time alerts will not be immediately available during the commissioning period.
-However, the incremental template generation and alert production processes will be tested and optimized during commissioning with LSSTCam, resulting in a set of prompt data products, including alerts.
-Broker teams will be given access to these alerts for development purposes as soon as they are produced to understand their characteristics and to help to validate their quality, rather than to enable rapid followup and Early Science.
-The Alert and Prompt Product databases will be made available for query prior to the DP2 data release.
+A variety of technical, scientific, and organizational criteria must be met prior to the release of the first public alerts from LSSTCam.
+The prerequisites, which include formal agency approval, are summarized in \citeds{RTN-061}.
+Accordingly, alert production is expected to begin around the time of the Science Validation Surveys with LSSTCam and the start of the full LSST survey.
```
Comment (leannep) [inline alertproduction.tex:62]: Perhaps add a statement about not being full-scale routine AP until Y2?

**Review by leannep — APPROVED** [review]

**Comment by mjuric** [comment]
Love it!

One suggestion: change "SSP Catalogs" to "Solar System Products". (I think it's more self-contained, & we don't define "SSP" anywhere in the doc).

For that same table, I'd propose to shade columns that are in Ops differently than those in commissioning (and mention that in the caption). This would make it easier visually easy to see when Ops start. Mock-up attached!
image

File: datapreview.tex [hunk 2]
```diff
@@ -66,6 +66,7 @@ \subsection{Data Preview 1}
 
 In both DP1 scenarios, the exact processing pipelines and data products are still under discussion.
 At minimum,  DP1 will deliver visit-level calibrated images and catalogs to enable initial studies of observational and instrumental effects. 
+Attribution of known Solar System Objects is also expected.
```
Comment (leannep) [inline datapreview.tex:69]: I think the wording is too strong  -- I would prefer to be clear that this is a stretch goal. For example, I would not want to delay the release of DP1 that contains visit-level calibrated images and catalogs in order to carry out atrribution of known SS Object (or any other data product).

Comment (leannep) [reply datapreview.tex:69]: Any delay to the release of DP1 has a knock on impact on delivering DP2 and DR1

**Review by leannep — CHANGES_REQUESTED** [review]

File: datapreview.tex (hunk 2 above)
Comment (mjuric) [reply datapreview.tex:69]: If we're not able to associate SolSys at DP1 time, that will indicate /serious/ trouble. This is functionality we basically have now. I'm comfortable promising it.

Comment (leannep) [reply datapreview.tex:69]: There are other data products that we would like to release, e.g. coadds, and if possible we will but I am not comfortable "promising" them.  We have a lot of functionality that work but is not yet tested on real LSST data, which is why I want to manage expectations conservatively for DP1.

Comment (leannep) [reply datapreview.tex:69]:
```suggestion
Additional data products, such as coadds or attribution of known Solar System Objects is a stretch goal for DP1"
```

File: tables/rubin_early_dr_scenario.tex [hunk 3]
```diff
@@ -3,44 +3,47 @@
...
 % PP SSP Catalogs   &   --  &   -- &    \mycirc[RubinDarkTeal]   &   --  &    \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]   &    \mycirc[RubinDarkTeal]  \\  \hline
-DRP SSP Catalogs   &   --  &   -- &    --  &   --  &   --  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  \\  
+Alerts &   --  &   -- &  --  &   --  & \mycirc[RubinDarkTeal] &  --  &  --  &  -- & --   \\ \hline 
+SSP Catalogs   &   --  &   -- &    \mycirc[RubinDarkTeal]  &   \mycirc[RubinDarkTeal]  & \mycirc[RubinDarkTeal] &  \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  &  \mycirc[RubinDarkTeal]  &    \mycirc[RubinDarkTeal]  \\  
  \arrayrulecolor{black}\hline
 \end{tabular}}
-\caption{Summary of Data Release data products expected in each data preview and early LSST data release.}
+\caption{Summary of data products expected in each early LSST data release.}
 \label{tab:data-preview-summary}
 \end{table}
```
Comment (leannep) [inline tables/rubin_early_dr_scenario.tex:49]: The rework of this table was intended to include only DRP data products, with another section to be added for PP data products. A Data Preview is made with DRP  ... AP and PP are additional components of the ES program.

File: datapreview.tex (hunk 2 above)
Comment (mjuric) [reply datapreview.tex:69]: Proposal: by mid-October we will have demonstrated known-object association with ComCam (and should be running it routinely). The level of effort needed for SolSys-in-DP1 should also be well understood at that point (should be just a few extra columns in the Source table).

If we manage to do that, would it be possible to add this type of language into RTN-011 (before Nov 1st)?

Comment (leannep) [reply datapreview.tex:69]: Thinking further .. aren't SSObjects identified as a part of DIA? We are not intending to make any DIA products available as part of DP1. DP1 is only single-epoch PVIs and catalogs of detections in those PVIs. There are no DIA products in DP1.

Comment (leannep) [reply datapreview.tex:69]: We do not yet know what the DP1 dataset will look like. The ComCam on-sky program must support the engineering activities as its first priority with science-grade observing as a second priority.  We cannot make promises that depend on our ability to observe or the duration of observing during ComCam.

Comment (leannep) [reply datapreview.tex:69]: Proposal:  In both DP1 scenarios, the exact processing pipelines and data products are still to be determined and depend strongly on the dataset that emerges from the commissioning on-sky observing program.
At minimum,  DP1 will deliver visit-level calibrated images and catalogs to enable initial studies of observational and instrumental effects.  Rubin will do its best to deliver the maximum of data products possible with the on-sky dataset, however at this time, all other data products described in the DPDD remain a stretch goal.

File: tables/rubin_early_dr_scenario.tex (hunk 3 above)
Comment (ebellm) [reply tables/rubin_early_dr_scenario.tex:49]: I understand the rationale for only including DRPs in the table--I just worry that it leads people on- and off-project to forget where AP comes in the sequence.  The new timeline figure later in the document helps some if we are sure to show it.

File: datapreview.tex (hunk 2 above)
Comment (ebellm) [reply datapreview.tex:69]: I don't object to that wording.

File: datapreview.tex [hunk 4]
```diff
@@ -66,6 +66,7 @@ \subsection{Data Preview 1}
 
 In both DP1 scenarios, the exact processing pipelines and data products are still under discussion.
```
Comment (leannep) [inline datapreview.tex:67]:
```suggestion
In both DP1 scenarios, the exact processing pipelines and data products are still to be determined and depend strongly on the dataset that emerges from the commissioning on-sky observing program.
```

File: datapreview.tex [hunk 5]
```diff
@@ -66,6 +66,7 @@ \subsection{Data Preview 1}
...
 At minimum,  DP1 will deliver visit-level calibrated images and catalogs to enable initial studies of observational and instrumental effects.
```
Comment (leannep) [inline datapreview.tex:68]: At minimum, DP1 will deliver visit-level calibrated images and catalogs to enable initial studies of observational and instrumental effects.

File: datapreview.tex (hunk 2 above)
Comment (leannep) [inline datapreview.tex:69]:
```suggestion
Rubin will do its best to deliver the maximum of data products possible with the commissioning on-sky dataset, however at this time, for DP1, all other data products described in the DPDD remain a stretch goal.
```

**Review by leannep — CHANGES_REQUESTED** [review]
Added additional suggestions as per our discussion

File: tables/rubin_early_dr_scenario.tex (hunk 3 above)
Comment (leannep) [reply tables/rubin_early_dr_scenario.tex:49]: Good point, let's merge this branch as you propose and we can then take a more global look at how we organise and present the data products

**Pull request merged**